packages/aura/src/typography.css

```diff
@@ -0,0 +1,101 @@
+@import './fonts/InstrumentSans/InstrumentSans.css';
+
+:where(html) {
+  --aura-font-family-system: system-ui, ui-sans-serif, sans-serif;
+  --aura-font-family-instrument-sans: 'Instrument Sans', var(--aura-font-family-system);
+
+  --aura-font-family: var(--aura-font-family-instrument-sans);
+
+  --aura-base-font-size: 16;
+  --aura-font-size-scale: 1.125;
+  --aura-base-line-height: 1.4;
+
+  --aura-font-weight: 400;
+  --aura-font-weight-medium: 500;
+  --aura-font-weight-semibold: 600;
+}
+
+@media (pointer: fine) {
+  :where(html) {
+    --aura-base-font-size: 14;
+  }
+}
+
+@supports not (-webkit-touch-callout: none) {
+  @media (pointer: coarse) {
+    /* Android optimizations */
+  }
+}
+
+@supports (-webkit-touch-callout: none) {
+  @media (pointer: coarse) {
+    :where(html) {
+      /* iOS/iPadOS optimizations */
+
+      /* Support iOS dynamic text size */
+      font: -apple-system-body;
```

Comment:
This needs to be cleared in SonarCloud, so that it doesn't produce a warning. We can't add another font family after the Apple one, e.g. `font: -apple-system-body, sans-serif` doesn't work as expected.